src/components/DocumentationTopic/PrimaryContent/DeclarationGroup.vue

```diff
@@ -162,7 +163,8 @@ export default {
       await this.$nextTick(); // wait for identifier to update
       this.isExpanded = false; // collapse the list
       await waitFor(500); // wait for animation to finish
-      this.$router.push(this.references[identifier].url);
```

Comment:
You could also do the following instead of using `buildUrl`:

```js
this.$router.push({
  path: this.references[identifier].url,
  query: this.$route.query,
});
```

(Just noting the other option—you don't need to change it)